public/locales/fr-FR/translations.json

```
@@ -0,0 +1,488 @@
+{
+  "action": "action",
+  "assets": "Actifs",
+  "assets.nft_tab_title": "Jetons non fongibles",
+  "assets.issued_tab_title": "Tokens émis",
+  "assets.no_issued_message": "Aucun token trouvé",
+  "assets.no_nfts_message": "Aucun NFT trouvé.",
+  "network": "Réseau",
+  "network_name": "Réseau inconnu",
+  "network_name_testnet": "Testnet",
+  "network_name_mainnet": "Mainnet",
+  "network_name_devnet": "Devnet",
+  "network_name_amm": "AMM-Devnet",
+  "network_name_hooks_testnet": "Hooks-Testnet",
+  "network_name_custom": "Personnalisé",
+  "app.meta.description": "Explorateur XRPL",
+  "app.meta.author": "Ripple",
+  "explorer": "Explorateur",
+  "xrpl_org": "XRPL.org",
+  "github": "GitHub",
+  "header.search.placeholder": "Rechercher par adresse, registre ou transaction",
+  "xrp": "XRP",
+  "xrpl_explorer": "Explorateur XRPL",
+  "ledgers": "Registres",
+  "ledger": "Registre",
+  "taxon": "Taxon",
+  "token_id": "Token ID",
+  "token_transactions": "Transactions du token",
+  "transactions": "Transactions",
+  "transaction_short": "TX",
+  "nodes": "Noeuds",
+  "validator": "Validateur",
+  "validators": "Validateurs",
+  "upgrade_status": "Statut des versions",
+  "version": "v{{number}}",
+  "component_error": "Une erreur s'est produite!",
+  "1H": "1H",
+  "24H": "24H",
+  "30D": "30J",
+  "total_transactions": "# de Txns",
+  "total_fees": "Frais Totaux",
+  "async_component_timedout": "Le chargement du composant a pris plus de temps que prévu",
+  "async_component_failed": "Impossible de charger le composant",
+  "account_not_found": "Compte non trouvé",
+  "account_empty_title": "Aucun id de compte n'a été fourni",
+  "account_empty_hint": "Saisissez un id de compte dans le champs de recherche",
+  "check_account_id": "Vérifiez votre ID de compte",
+  "accounts.xrp_balance": "Solde XRP",
+  "accounts.other_balances": "Autres soldes",
+  "accounts.other_balances_short": "Soldes",
+  "amount": "Montant",
+  "currency_code": "Code FX",
+  "currency_balance": "Solde {{currency}}",
```

```suggestion
  "currency_balance": "Solde <0>{{currency}}</0>",
```

This will fix the interpolation of this page https://livenet.xrpl.org/accounts/rrhrqxRPp79eVGBh84w1veWQKrZPEjipcz/assets